src/visumorph/flipping.py

```diff
@@ -0,0 +1,19 @@
+def flip(input_file, v=0):
+    """Load image from an image file and return as an image numpy array.
+
+    Parameters
+    ----------
+    input_file : str
+        Path to image file.
+    v : int, optional
+        Flip vertically if v=1, horizontally if v=0. By default, v=0.
+
+    Returns
+    -------
+    df
```

Comment:
I think we need to specify of type of returned value here as specified by the [numpy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#returns)

As name for returned value is optional, I would suggest we don't specify it for this moment for greater flexibility. Please also note that `df` is usually associated with variable of type `pandas.DataFrame` by convention.
```suggestion
    ndarray
```